components/ai_chat/core/browser/conversation_driver_unittest.cc

```diff
@@ -811,4 +848,203 @@ TEST_F(ConversationDriverUnitTest, ModifyConversation) {
             "answer2");
 }
 
+class PageContentRefineTest : public ConversationDriverUnitTest,
+                              public testing::WithParamInterface<bool> {
+ public:
+  void SetUp() override {
+    ConversationDriverUnitTest::SetUp();
+    scoped_feature_list_.InitWithFeatureState(features::kPageContentRefine,
+                                              GetParam());
+    embedder_task_runner_ = base::ThreadPool::CreateSequencedTaskRunner(
+        {base::MayBlock(), base::TaskPriority::BEST_EFFORT});
```

Comment:
same here

Reply:
fixed in 6ccf528315ebf8400be0bf904c07f7ec4b0a58d4